WordPress/src/main/java/org/wordpress/android/ui/comments/CommentActions.java

```diff
@@ -437,7 +437,8 @@ public void run() {
      */
     private static void deleteComment(final int accountId,
                                         final Comment comment,
-                                        final CommentActionListener actionListener) {
+                                        final CommentActionListener actionListener,
+                                        final boolean forceDelete) {
```

Comment:
:100: was trying to follow along the names used on the XMLRPC API but it makes perfect sense from the CommentActions class point of view :+1: